reception/reception.py

```diff
@@ -1,37 +1,35 @@
 '''    
     Program that checks if name exist in list
 '''
+print("\n\tRegistration Checker \n")
 
-# capture List of string 
+def read_file(file_to_read):
+    # read from file
+    names_list = []
+    f = open(file_to_read,'r')
+    NamesList = f.readlines()      # Using .readlines()
+    f.close()
+    for name in NamesList:
+        names_list.append((name.title()).rstrip('\n'))
 
-ordinary_list = []
-f = open('ordinary_list.txt','r')
-ordinary = f.readlines()      # Using .readlines()
-f.close()
-for name in ordinary:
-    ordinary_list.append((name.title()).rstrip('\n')) 
-# print(ordinary_list)
+    return names_list
 
-vip_list = []
-f = open('vip_list.txt','r')
-vip = f.readlines()      # Using .readlines()
-f.close()
-for name in vip:
-    vip_list.append((name.title()).rstrip('\n')) 
-# print(vip_list)
+# print(read_file('ordinary_list.txt'))
+# print(read_file('vip_list.txt'))
 
-User_Name = (input("Please Enter your Name:\n > ")).title()
-# Ask a user to enter one of their names and check through the lists to see
-# if any(User_Name in mystring for mystring in vip_list) == True:
-#     print ("\n".join(names for names in vip_list if User_Name.title() in names))
-
-def registration_checker(User_Name):
-    if any(User_Name in mystring for mystring in vip_list) == True:
+def registration_checker():
+    User_Name = (input("Please Enter your Name:\n > ")).title()
+    # Ask a user to enter one of their names and check through the lists to see
+    ordinary_list = read_file('ordinary_list.txt')
+    vip_list = read_file('vip_list.txt')
+    if any(User_Name in mystring for mystring in vip_list) == any(User_Name in mystring for mystring in ordinary_list):
+        print ("\n".join(names for names in vip_list if User_Name.title() in names) + ", VIP")
+        print ("\n" .join(names for names in ordinary_list if User_Name.title() in names) + ", Ordinary")
+    elif any(User_Name in mystring for mystring in vip_list) == True:
```

Comment:
u have un necesarily over nested the if ...i know theres a better way to summarise it up with a few ifs